mod_reforged/hooks/skills/actives/barbarian_fury_skill.nut

```diff
@@ -4,5 +4,6 @@
 		__original();
 		this.m.Icon = "skills/active_175.png";
 		this.m.IconDisabled = "skills/active_175_sw.png";
+		this.m.Description = "Switch places with another character directly adjacent, provided neither the target is stunned or rooted, nor the character using the skill is. Rotate the battle line to keep fresh troops in front!"; // Vanilla is missing a description for this skill
```

Comment:
I prefer moving `adjacent` forward and using `ally` instead of character.
`[...] character directly adjacent, [...]` reads weird.

### Proposal
`Switch places with an adjacent ally, [...]`

Reply:
I used the vanilla rotation description, but I agree your proposal is better.